events/importer/base.py

```diff
@@ -60,11 +60,22 @@ def __init__(self, options):
             self.bounding_box = None
         self.gps_to_target_ct = CoordTransform(gps_srs, target_srs)
 
-        self.setup()
+        # Pre processing phase runs your importers setup.
+        # You can override this check by setting 'ignore_setup' = True into your importer.
 
-        # this has to be run after setup, as it relies on organization and data source being set
-        self._images = {obj.url: obj for obj in Image.objects.filter(publisher=self.organization,
-                                                                     data_source=self.data_source)}
+        ignore_setup_phase = getattr(self, 'ignore_setup', False)
+
+        if ignore_setup_phase == False:
```

Comment:
[Don’t compare boolean values to True or False using ==](https://peps.python.org/pep-0008/)

Reply:
Should be fixed